packages/features/webhooks/lib/cron.ts

```diff
@@ -25,19 +25,13 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   // run jobs
   for (const job of jobsToRun) {
-    // Fetch the webhook configuration so that we can get the secret.
-    const [appId, subscriberId] = job.jobName.split("_");
-    const webhook = await prisma.webhook.findUniqueOrThrow({
-      where: { id: subscriberId, appId: appId !== "null" ? appId : null },
-    });
     try {
```

Comment:
Is this intended?

Reply:
Good catch, had some weird git issues so these must have snuck in.